e2e_workflow/settings.xml

```diff
@@ -0,0 +1,32 @@
+<settings
```

Comment:
Settings file used when serving war with tomcat

Reply:
Maybe add that as a comment in this XML file